src/main/java/frc/robot/subsystems/CoralManipulator.java

```diff
@@ -74,12 +80,18 @@ public Command stopRollers() {
     }
 
     public boolean hasCoral() {
-        return !beambreak.get(); // get() returns true when circuit is closed
+        return elevBeambreak.get() && !beambreak.get(); // get() returns true when circuit is closed
+    }
+
+
+    public boolean coralInputted(){
+        return !elevBeambreak.get();
     }
 
     @Override
     public void periodic() {
-        beambreakPub.set(hasCoral());
+        beambreakPub.set(!beambreak.get());
+        elevBeambreakPub.set(!elevBeambreak.get());
```

Comment:
either change these to not be inverted or change the name to represent what it means